sdk/ai/azure-ai-inference/README.md

```diff
@@ -0,0 +1,476 @@
+Serverless API endpoints and Managed Compute Endpoints (formerly known as Managed Online Endpoints). The client library makes services calls using REST API version `2024-05-01-preview`, as documented in [Azure AI Model Inference API](https://learn.microsoft.com/azure/ai-studio/reference/reference-model-inference-api). For more information see [Overview: Deploy models, flows, and web apps with Azure AI Studio](https://learn.microsoft.com/azure/ai-studio/concepts/deployments-overview).
+
+Use the model inference client library to:
+
+* Authenticate against the service
+* Get information about the model
+* Do chat completions
+* Get text embeddings
+<!-- * Get image embeddings -->
+
+Note that for inference using OpenAI models hosted on Azure, you should be using the official [OpenAI Python client library](https://github.com/openai/openai-python) in product code instead of this client. However, for development and evaluation purposes (comparing OpenAI models to other models in the Azure AI Studio catalog), you can use the azure-ai-inference Python client library with Azure OpenAI endpoints, as shown [in this sample](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/ai/azure-ai-inference/samples/sample_chat_completions_azure_openai.py).
+
+[Product documentation](https://learn.microsoft.com/azure/ai-studio/reference/reference-model-inference-api)
+| [Samples](https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/ai/azure-ai-inference/samples)
+| [API reference documentation](https://aka.ms/azsdk/azure-ai-inference/python/reference)
+| [Package (Pypi)](https://aka.ms/azsdk/azure-ai-inference/python/package)
+| [SDK source code](https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/ai/azure-ai-inference/azure/ai/inference)
+
+## Getting started
+
+### Prerequisites
+
+* [Python 3.8](https://www.python.org/) or later installed, including [pip](https://pip.pypa.io/en/stable/).
+* An [Azure subscription](https://azure.microsoft.com/free).
+* An [AI Model from the catalog](https://ai.azure.com/explore/models) deployed through Azure AI Studio.
+* To construct the client library, you will need to pass in the endpoint URL. The endpoint URL has the form `https://your-deployment-name.your-azure-region.inference.ai.azure.com`, where `your-deployment-name` is your unique model deployment name and `your-azure-region` is the Azure region where the model is deployed (e.g. `eastus2`).
+* Depending on your model deployment and authentication preference, you either need a key to authenticate against the service, or Entra ID credentials. The key is a 32-character string.
+
+### Install the package
+
+To install the Azure AI Inferencing package use the following command:
+
+```bash
+pip install azure-ai-inference
+```
+
+To update an existing installation of the package, use:
+
+```bash
+pip install --upgrade azure-ai-inference
+```
+
+## Key concepts
+
+### Create and authenticate a client directly, using key
```

Comment:
Samples should use entra id. Key based auth is not to be promoted per Microsoft guidelines.

Reply:
EntraID is not yet supported on MaaS. Once it's supported, and I make another release, I will switch most samples to use EntraID.